logger/src/lib.rs

```diff
@@ -54,6 +57,27 @@ pub fn setup() {
     setup_with_default("error");
 }
 
+// When we have multi-threading tests like local-clusters, it's nice to print
+// the thread id at the beginning, so you can filter out logs belonging to
+// different threads.
+pub fn setup_logging_with_thread_id() {
```

Comment:
Currently `setup_with_default()` is the most commonly used method in local cluster, should this new function take a `filter: &str` as well?